HOWTO

```diff
@@ -2816,7 +2816,7 @@ Threads, processes and job synchronization
 
 	By default, fio will continue running all other jobs when one job finishes
 	but sometimes this is not the desired action.  Setting ``exitall`` will
-	instead make fio terminate all other jobs when one job finishes.
+	instead make fio terminate all other jobs in a group when one job finishes.
```

Comment:
Probably should make that a bit more precise since we're changing it - rather than "in a group", it should be "[...] in the same group, then one job that group finishes."

Reply:
Yea, you're right. I'll fix it tomorrow.

Would you be interested in an additional option ``exitgroup``, with the same semantics as exitall now has. And then having ``exitall`` actually exit all workers?

Reply:
> Would you be interested in an additional option exitgroup, with the same semantics as exitall now has. And then having exitall actually exit all workers?

We can't break existing use cases by changing the meaning of it now, if you want that kind of behavior then we'd need a new option.

Reply:
Hm. True …
I've actually already hacked a version up … (because I want to have grouping for log aggregation of only some jobs and still be able to quit after a specfic job has finished)

Do you have a preference for naming this new option? I can only come up with ``really_exitall`` ;)

Reply:
`exit_every_group`?

Reply:
❤️